midend/expr_uses.h

```diff
@@ -27,6 +27,9 @@ class exprUses : public Inspector {
     bool preorder(const IR::Path *p) override {
         if (p->name == look_for) result = true;
         return !result; }
+    bool preorder(const IR::Member *m) override {
+        if (m->toString() == look_for) result = true;
```

Comment:
Do not use expression->toString() in the compiler; it is only meant for displaying information to the user. It is lossy.

What is the purpose of this commit? Who uses this method and why?